static/app/components/tabs/tab.tsx

```diff
@@ -37,64 +42,165 @@ function handleLinkClick(e: React.PointerEvent<HTMLAnchorElement>) {
   }
 }
 
+interface BaseTabProps {
+  children: React.ReactNode;
+  hidden: boolean;
+  isSelected: boolean;
+  orientation: Orientation;
+  overflowing: boolean;
+  tabProps: DOMAttributes<FocusableElement>;
+  /**
+   * Additional props to be merged with `tabProps`. This is used
+   * by <DraggableTab> to pass in props used for drag-and-drop functionality.
+   */
+  additionalProps?: React.HTMLAttributes<HTMLElement>;
+  borderStyle?: 'solid' | 'dashed';
+  to?: string;
+  variant?: 'vanilla' | 'draggable';
+}
+
+export const BaseTab = forwardRef(
```

Comment:
This `<BaseTab/>` component is functionally exactly the same the previous `<BaseTab/>` comopnent, except its being wrapped by a new `<Tab/>` component.